packages/block-editor/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 -   Move component styles needed for iframes to content styles ([#47103](https://github.com/WordPress/gutenberg/pull/47103)).
 -   Block Inserter: Correctly apply style to the default inserter ([#47166](https://github.com/WordPress/gutenberg/pull/47166)).
 -   List View: Fix crash when the first template-parts is deleted width del key ([#47227](https://github.com/WordPress/gutenberg/pull/47227)).
+-   Block Preview: Try to fix multiple rendering of controls ([#47331](https://github.com/WordPress/gutenberg/pull/47331)).
```

Comment:
Probably change the message here to explain that you disabled the ability select text in the `useBlockPreview` hook.